cli/tests/test_cli.py

```
@@ -0,0 +1,1491 @@
+                        with patch('requests.post') as mock_post:
+                            # Mock API error response
+                            mock_response = MagicMock()
+                            mock_response.status_code = 400
+                            mock_response.json.return_value = {"error": "Validation failed"}
+                            mock_post.return_value = mock_response
+
+                            runner = CliRunner()
+                            result = runner.invoke(import_evidences, ['--file', 'test.csv'])
+
+                            # Verify error handling
+                            captured = capsys.readouterr()
```

Ruff (0.11.9)

Local variable `captured` is assigned to but never used

Remove assignment to unused variable `captured`

(F841)